Sources/Fluent/Query/QueryField.swift

```diff
@@ -43,21 +43,33 @@ extension QueryField: ExpressibleByStringLiteral {
 
 extension Dictionary where Key == QueryField {
     /// Accesses the _first_ value from this dictionary with a matching field name.
-    public func firstValue(forField fieldName: String) -> Value? {
+    public func firstValue(forField fieldName: String, on entity: String) -> Value? {
```

Comment:
this "firstValue" should stay the same (no on entity label). Use `value(forEntity entity: String, atField field: String)` instead. A `QueryField` with same table name _and_ entity is the same query field, so it doesn't make sense to say "first" value for a specific field name and entity--it's the "only value"